styles.css

```diff
@@ -1001,3 +987,444 @@ h1 {
+    right: 154px;
+    top: calc(40% - 40px / 2);
+    color: white;
+    gap: 10px;
+  }
+
+  .headline {
+    position: absolute;
+    width: 100%;
+    height: 870px;
+    left: 0;
+    top: 72px;
+    background: #1c1a19;
+    background-image: url(Images/bg-d.png);
+    background-repeat: no-repeat;
+    background-position: right top;
+    background-size: 553px 769.27px;
+  }
+
+  .headline-2 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 10px;
+    position: absolute;
+    width: 566px;
+    height: 739px;
+    left: 143px;
+    top: 52px;
+  }
+
+  h1 {
+    width: 566px;
+    height: 160px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .link {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    width: 20px;
+    height: 98.96px;
+    left: 36px;
+    top: 234px;
+    gap: 12px;
+  }
+
+  .supporting-text {
+    width: 566px;
+    height: 24px;
+    display: flex;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .description {
+    width: 566px;
+    height: 24px;
+    display: flex;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  .recent-works {
+    width: 100%;
+    height: 1805px;
+    top: 870px;
+  }
+
+  .title {
+    display: flex;
+    flex-direction: row;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .head-title {
+    display: inline-flex;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    width: 315px;
+    height: 52px;
+    padding: 0 10px 0 0;
+    margin-left: 100px;
+  }
+
+  .line {
+    width: 794px;
+    height: 0;
+    border: 1px solid #212121;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    top: 157px;
+    align-self: center;
+  }
+
+  .sub-line {
+    display: none;
+  }
+
+  .featured-image {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    position: absolute;
+    width: 1156px;
+    height: 448px;
+    left: -153px;
+    top: 158px;
+  }
+
+  .img {
+    width: 484px;
+    height: 448px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    background-image: url(Images/Img-Placeholder.png);
+    background-repeat: no-repeat;
+    margin: 34px 0 0 164px;
+    background-size: 400px 380px;
+  }
+
+  .right-block {
+    width: 448px;
+    height: 348px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 24px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .title-post {
+    width: 448px;
+    height: 48px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin-top: 55px;
+  }
+
+  .sup {
+    width: 448px;
+    height: 100px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .tags {
+    margin-top: 0;
+  }
+
+  .card-works {
+    position: absolute;
+    width: 369px;
+    height: 435px;
+    left: 55px;
+    top: 659px;
+    border: 2px solid #091e42;
+    background-image: url(/Images/img2.png);
+    background-position: 100% 100%;
+  }

```

Comment:
- [ ] Kindly fix your background images, since were you are doing a great job showing them on desktop version. On mobile version are not present now.
![image](https://user-images.githubusercontent.com/93630700/208179816-9d7ef2a3-3728-4b04-b6f3-b55faafcb104.png)


- [ ] Kindly check your desktop version, all the jobs cards are not displayed on this version

![image](https://user-images.githubusercontent.com/93630700/208179889-cdc709b0-ac3c-4691-9f88-2abd4cd46349.png)

Reply:
hello bregorn, i have tried cloning and the images works fine....please try reviewing it again  @BregornOriginal and the other changes i am going to be working on